live/client.py

```diff
@@ -132,3 +134,58 @@ def stop_event(self, event_id):
         # Send request and do exception handling
         self.send_request(http_method="POST", url=url,
                           headers=headers, body=body)
+
+    def find_devices_in_use(self):
+        events_url = f'{self.server_ip}/live_events'
+        events_headers = self.generate_headers(events_url)
+        events_xml = self.send_request(
+            http_method="GET", url=events_url, headers=events_headers)
+
+        events_list = ET.fromstring(events_xml.text)
+
+        # Find in use devices from all events
+        in_use_devices = set()
+        for event in events_list:
+            status = event.find('status').text
+            for input in event.findall('input'):
```

Comment:
`input` is a reserved word for an internal method, it would be good to change this variable name